book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,472 @@
+#### Plain Language Summaries
+If your event has a large focus on research content, consider writing plain language summaries of core content so that people with lived experience who do not have a background in research are able to engage in the sessions and understand the projects that are discussed. 
+
+Please refer to The Turing Way's chapter on {ref}`cm-lay-summaries` for more guidance, and you can view good examples of plain language summaries from the [AI for Multiple Long Term Conditions research projects](https://aim-rsf.github.io/Glossary-of-Terms/pages/aim-projects.html). 
+
+(cl-inclusive-events-comms-accessibility-guide)=
+## Content Accessibility Guidance 
+Ensuring that event communications, including printed content, are accessible to all attendees is critical. We have written a short guide for creating accessible content and presentations in {ref}(cl-inclusive-events-accessibility-guide) where you will find in-depth information and suggestions for how to make your presentations and content more accessible for your event.
+
+These recommendations include using at least a size 14 text in an accessible font type, and ensuring that your presentations and other material use high-contrast colours to support readability. You should also ensure that any images you use are accompanied by an alternative text description and remember to not embed any important information in an image or image file. 
+
+A large part of inclusion is about building trust. 
+Even if your email contains all the details that an image of your flyer does, a person who is unable to access the information in the image will not know if there is information they are missing out on. 
+Try to use multiple channels (such as a PDF flyer with alt-text, plain text email, accessible website, and a captioned video) to duplicate the communication of core information and engage with the communities of people with lived experience to understand how to communicate best with them. 
+
+Please see the {ref}(cl-inclusive-events-accessibility-guide) for more information, and you can find recommendations for other specific content considerations in a [blog post](https://accessibility.blog.gov.uk/2016/09/02/dos-and-donts-on-designing-for-accessibility/) published by the Accessibility Team of the UK's Home Office Digital Group. 
+These posters make recommendations for how to make communications and information accessible for people with: 
+- low vision
+- D/deaf and hard of hearing
+- dyslexia
+- motor disabilities
+- users on the autistic spectrum
+- users of screen readers
+
+
+(cl-inclusive-events-comms-vendor-timelines)=
+### Working with Vendors 
+As part of deciding what event and promotional materials will be required, identify who will develop these materials, whether it’s an external marketing agency or an internal graphic designer, and if there are additional requirements that vendors may have, such as specific printers, lead times, and file format preferences. 
```

Comment:
```suggestion
As part of deciding what event and promotional materials will be required, identify who will develop these materials. 
This can be an external marketing agency or an internal graphic designer. 
There  may be additional requirements that vendors may have, such as specific printers, lead times, and file format preferences. 
```
Or something else to shorten/split this sentence

Reply:
I think the split you suggested is great and makes it a lot easier to read.